src/app/components/LiveLabel/index.tsx

```diff
@@ -34,14 +54,14 @@ const LiveLabel = ({
         dir={dir}
         {...(ariaHidden && { 'aria-hidden': 'true' })}
       >
-        {`${liveText} `}
+        {`${liveLabel} `}
```

Comment:
Does this work, or do we still need the space after the label?

```suggestion
        {liveLabel}
```

Reply:
I think we need to keep the space after the label, so ignore this 😸